frontend/appflowy_tauri/src/appflowy_app/components/grid/GridTableCount/GridTableCount.tsx

```diff
@@ -1,7 +1,9 @@
-import { useGridTableCount } from './GridTableCount.hooks';
+import { useDatabase } from '$app/components/_shared/database-hooks/useDatabase';
+import { ViewLayoutPB } from '@/services/backend';
 
-export const GridTableCount = () => {
-  const { count } = useGridTableCount();
+export const GridTableCount = ({ viewId }: { viewId: string }) => {
```

Comment:
I would suggest just sending `row.length` from the parent, instead of calling `useDatabase` again.
```suggestion
export const GridTableCount = ({ rowCount }: { rowCount: number }) => {
```

Reply:
True. I guess there's no need to access the database separately. 
Did you want me to keep [GridTableCount.tsx](https://github.com/AppFlowy-IO/AppFlowy/pull/2973/files#diff-f5d814cc58dde9e6b97e0f0c9ac1a00726859c958ddd63e5e3a7f2723e7ed5ab) and [GridTableCount.hooks.ts](https://github.com/AppFlowy-IO/AppFlowy/pull/2973/files#diff-ce86ab22b922daf9f734414defdd78a7f3ec6e5804d292ca150914d60e81f1cd) though? It's such a small chunk of code that I wasn't sure if it's worth maintaining a separate component and a hook. I can easily retain them and pass `row.length` down to [GridTableCount.tsx](https://github.com/AppFlowy-IO/AppFlowy/pull/2973/files#diff-f5d814cc58dde9e6b97e0f0c9ac1a00726859c958ddd63e5e3a7f2723e7ed5ab) as well.

Reply:
In fact, it was drawn out because of this requirement, but it’s okay to follow your way of writing for the time being
<img width="311" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy/assets/108015703/6e3c3f9e-0404-4cf4-b749-33a5af7e43a7">

Reply:
@qinluhe Oh interesting. that makes sense. Ok then, it's probably not necessary to store the row information in the Redux store or access the database in the GridTableCount component using a hook; however, I should probably segregate the row filtering logic in [GridTableCount.tsx](https://github.com/AppFlowy-IO/AppFlowy/pull/2973/files#diff-f5d814cc58dde9e6b97e0f0c9ac1a00726859c958ddd63e5e3a7f2723e7ed5ab).
I'll retain [GridTableCount.tsx](https://github.com/AppFlowy-IO/AppFlowy/pull/2973/files#diff-f5d814cc58dde9e6b97e0f0c9ac1a00726859c958ddd63e5e3a7f2723e7ed5ab) so we can easily implement this count feature in the future. I just updated the PR, so let me know what you think.